Dockerfile

```diff
@@ -1,5 +1,10 @@
 FROM python:3.9.13-slim-buster
 
+RUN apt-get update && apt-get install -y \
+    # required for ./cookiecutter-docker.sh -t <git_repo>
+    git \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
Looking back at the code, I wonder if we should just remove the `-t` template option all together, rather than adding git support. 

Reason being, the docker image is tightly coupled to this specific project, and you already need to have cloned the repo locally to build the image in the first place. Given that, I'm not sure it would actually make sense to provide anything other than the default value.